src/styles/main.scss

```diff
@@ -91,21 +112,39 @@ header {
 .fast {
   width: 200px;
 }
+
 .closeBtn {
   width: 20px;
   margin-top: 120px;
   align-self: flex-end;
 }
+
 .cm {
   display: flex;
   flex-direction: column;
   align-items: flex-start;
   height: 200px;
   justify-content: space-around;
 }
+
 footer {
-  margin-top: auto;
+  font-size: 20px;
+  height: 5%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: #402e27;
+  position: fixed;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  color: #c9c9c9;
 }
+
 .insights {
   height: 50%;
```

Comment:
- [ ] It would be better to fix the linter errors, as I am performing the code review I see you fixing it. Kindly fix it before submitting the next code review otherwise it will interfere with the code review :man_technologist:

Reply:
Linters errors has been fixed.